lib/src/Base/Stat/IsotropicCovarianceModel.cxx

```diff
@@ -0,0 +1,219 @@
+//                                               -*- C++ -*-
+/**
+ *
+ *  Copyright 2005-2021 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include "openturns/IsotropicCovarianceModel.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+CLASSNAMEINIT(IsotropicCovarianceModel)
+
+
+/* Parameters constructor */
+IsotropicCovarianceModel::IsotropicCovarianceModel(const CovarianceModel & oneDimensional,
+  const UnsignedInteger inputDimension)
+  : CovarianceModelImplementation()
```

Comment:
better use the CovarianceModelImplementation(scale, amplitude) ctor instead setting them after

Reply:
CovarianceModelImplementation(oneDimensional.getScale(), oneDimensional.getAmplitude()

Reply:
Supprimer setInputDimension, setScale et setAmplitude

Reply:
CovarianceModelImplementation(inputDimension)

Reply:
And the method CovarianceModelImplementation::setInputDimension should then be dropped.